TrueDialog.Client.Standard.WebTests/WeatherForecast.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace TrueDialog.Client.Standard.WebTests
+{
+    public class WeatherForecast
+    {
+        public DateTime Date { get; set; }
+
+        public int TemperatureC { get; set; }
+
+        public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
```

Comment:
(Odd that they chose to divide instead of multiply here: 32 + TempC * 1.8)

ANYHOW, this looks like leftovers from the sample controller that Microsoft adds?  Should probably remove this.